core/templates/components/state-directives/answer-group-editor/answer-group-editor.directive.ts

```diff
@@ -310,7 +310,6 @@ angular.module('oppia').directive('answerGroupEditor', [
                   if (ctrl.isRuleEditorOpen()) {
                     ctrl.saveRules();
                   }
-                  $scope.$broadcast('updateAnswerGroupInteractionId');
```

Comment:
Should there be a replacement for this?

Reply:
Same as https://github.com/oppia/oppia/pull/10395#discussion_r474325816